src/views/Contacto.vue

```diff
@@ -3,7 +3,7 @@
     <div class="contacto">
       <h1 class="contacto__title">Contáctanos</h1>
       <div class="contacto__content-wrapper">
-        <contact-form></contact-form>
+        <contact-form :cause="this.$route.query.cause"></contact-form>
```

Comment:
Please change "cause" to "reason". "Cause" is not the best word in this context.